docs/cli-reference.md

```diff
@@ -212,6 +212,9 @@ Catalog related operations
   # Create new Node.js component with source from remote git repository.
   odo create nodejs --git https://github.com/openshift/nodejs-ex.git
 
+  # Create a new Node.js component with source from remote git repository using a branch, tag or commit ref.
+  odo create nodejs --git https://github.com/openshift/nodejs-ex.git --ref master
```

Comment:
If I remember correctly, this is generated from command help so could you please add it there and generate it to this doc? But let me just confirm the same

Reply:
Ah yes, forgot about that. Also, I'd use an example with a different `ref` than `master` since that's the default.

Reply:
Please autogenerate it

Reply:
ack, generated. also @metacosm  the nodejs-ex doesn't have a branch that is working hence I added an example for release `v1.0.1`